modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -1771,17 +1777,21 @@ public ObjectEntry updateStatus(
 			}
 		}
 		else {
-			ObjectDefinition objectDefinition =
-				_objectDefinitionPersistence.fetchByPrimaryKey(
-					objectEntry.getObjectDefinitionId());
-
 			_assetEntryLocalService.updateEntry(
 				objectDefinition.getClassName(), objectEntry.getObjectEntryId(),
 				null, null, true, objectEntry.isApproved());
 		}
 
 		_reindex(objectEntry);
 
+		if (!ObjectActionThreadLocal.isSkipObjectActionExecution()) {
```

Comment:
Why do we have to consider `isSkipObjectActionExecution` here and not in other places that use the same function in this same class like [here](https://github.com/liferay/liferay-portal/blob/1f99ce880f099caecc1e83d807d2f64eb2b00a88/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java#L390) and [here](https://github.com/liferay/liferay-portal/blob/1f99ce880f099caecc1e83d807d2f64eb2b00a88/modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java#L1610)?